candi/bashible/common-steps/node-group/000_add_node_users.sh.tpl

```diff
@@ -23,6 +23,105 @@ if bb-flag? disruption && bb-flag? reboot; then
   exit 0
 fi
 
+function error_check() {
+  error_code=$?
+  (( error_code != 0 ))
+}
+
+function kubectl_exec() {
+  kubectl --request-timeout 60s --kubeconfig=/etc/kubernetes/kubelet.conf patch nodeusers.deckhouse.io "${1}" "${2}" "${3}" "${4}"
+}
+
+# $1 - username $2 - request data
+function nodeuser_patch() {
```

Comment:
think about move it in bashbooster. Looks like we can reuse this function in multiple locations